ui/helpers/displayHelper.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I think this `eslint-disable-next-line` shouldn't be needed any more? :-)

Reply:
D'oh!  Thanks.  :)